### PR TITLE
Add `use-first-commit-message-for-new-prs` feature

### LINKED ISSUE #3329: Use first commit for PR title and description
If your branch has just one commit, then GitHub will use that commit message as the PR title and description, which is nice.

However, if your branch has more than one commit, then GitHub uses the branch name as the PR title and leaves the description blank.

I would rather GitHub always use the first commit in a PR branch as the PR title and description.

Example URL: https://github.com/zachwhaley/squashed-merge-message/compare/pr-title-example?expand=1

Screenshots:

Current:
Screen Shot 2020-07-09 at 11 03 29 AM

Desired:
Screen Shot 2020-07-09 at 11 03 59 AM

### PR DESCRIPTION
**Pull request by yakov116**
1. LINKED ISSUES:
   Closes #3329

2. TEST URLS:
	// Cross fork
   https://github.com/sindresorhus/refined-github/compare/master...yakov116:only-showing-the-branch-name?expand=1
	// Same repository
https://github.com/sindresorhus/refined-github/compare/fregante-patch-1...master?expand=1
Existing pr (will not run)
https://github.com/sindresorhus/refined-github/compare/master...yakov116:sync-new-pull-request-commit-message?expand=1

3. SCREENSHOT:
   image

### CODE REVIEW

File: source/features/use-first-commit-message-for-new-prs.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import * as textFieldEdit from 'text-field-edit';
+
+import features from '.';
+import {looseParseInt} from '../github-helpers';
```
Comment (fregante) [inline source/features/use-first-commit-message-for-new-prs.tsx:7]: Can you move this out of github-helpers in the other PR? It's a generic helper

Comment (yakov116) [reply source/features/use-first-commit-message-for-new-prs.tsx:7]: Done

**Comment by yakov116** [comment]
@fregante can you review this?

**Comment by kidonng** [comment]
I wonder why it's still erroring 😕

image

**Comment by yakov116** [comment]
Your on master?

Also this is the wrong pr 😆

**Pull request merged**